truss/cli/cli.py

```diff
@@ -237,6 +247,147 @@ def watch(
     remote_provider.sync_truss_to_dev_version_by_name(model_name, target_directory)
 
 
+def _create_chains_table(service) -> Tuple[rich.table.Table, List[str]]:
+    """Creates a status table e.g.
+
+                                                     Deployments
+    ╭──────────────────────┬──────────────────────┬────────────────────────────╮
+    │ Status               │ Chainlet             │                   Logs URL │
+    ├──────────────────────┼──────────────────────┼────────────────────────────┤
+    │  👾 DEPLOYING        │ SplitText            │ https://app.baseten.co/... │
+    │  👾 DEPLOYING        │ GenerateData         │ https://app.baseten.co/... │
+    │  👾 DEPLOYING        │ MistralLLM           │ https://app.baseten.co/... │
+    │  👾 DEPLOYING        │ TextToNum            │ https://app.baseten.co/... │
+    │  👾 DEPLOYING        │ Chain                │ https://app.baseten.co/... │
+    ╰──────────────────────┴──────────────────────┴────────────────────────────╯
+    """
+    table = rich.table.Table(
+        show_header=True,
+        header_style="bold yellow",
+        title=f"⛓️   {service.name} - Chainlets  ⛓️",
+        box=rich.table.box.ROUNDED,
+        border_style="blue",
+    )
+    table.add_column("Status", style="dim", min_width=20)
+    table.add_column("Name", min_width=20)
+    table.add_column("Logs URL")
+    statuses = []
+    for name, status, logs_url in service.get_info():
+        if status == ACTIVE_STATUS:
+            spinner_name = "active"
+        elif status in DEPLOYING_STATUSES:
+            spinner_name = "deploying"
+        else:
+            spinner_name = "failed"
+        spinner = rich.spinner.Spinner(spinner_name, text=status)
+        table.add_row(spinner, name, logs_url)
+        statuses.append(status)
+    return table, statuses
+
+
+@chain.command()  # type: ignore
+@click.argument("source", type=Path, required=True)
+@click.argument("entrypoint", type=str, required=True)
+@click.option(
+    "--name",
+    type=str,
+    required=False,
+    help="Name of the chain to be deployed, if not given, the entrypoint name is used.",
+)
+@click.option(
+    "--publish",
+    type=bool,
+    default=False,
+    is_flag=True,
+    help="Create chainlets as published deployments.",
+)
+@click.option(
+    "--promote",
+    type=bool,
+    default=False,
+    is_flag=True,
+    help="Replace production chainlets with newly deployed chainlets.",
+)
+@click.option(
```

Comment:
Is there a way to toggle this off at all? I don't think we _need_ to for now, but it feels like with "is_flag=True" and "default=True" that there would be no way to toggle it off (typically for is_flag=True, default is False).

All to say maybe remove this option for now, and add something to toggle it off if people ask for it (like --no-wait)